api/deploy/ocp/scc_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ocp
+
+import (
+	"testing"
+
+	secv1 "github.com/openshift/api/security/v1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewSecurityContextConstraints(t *testing.T) {
+	var scc *secv1.SecurityContextConstraints
+
+	getSCC := func() {
+		scc = NewSecurityContextConstraints(SecurityContextConstraintsDefaults)
+	}
+
+	require.NotPanics(t, getSCC)
+	require.NotNil(t, scc)
```

Comment:
How about adding more validation? Especially for `users` and when a deployer is passed.

Reply:
Sure, this is just to get started :smiley: